specification/immunisation-history.yaml

```diff
@@ -14,23 +14,25 @@ info:
     email: api.management@nhs.net
   description: |
     ## Overview
-    Use this API to access a patient’s immunisation history.
-
+    Use this API to access a patient's immunisation history.
+    
     You can:
-    - get a patient's coronavirus (COVID-19) immunisation history, based on their NHS number
+
+    - get a patient's coronavirus (`COVID-19`) immunisation history, based on their NHS number
+    - get a patient's influenza (`FLU`) immunisation history, based on their NHS number
+    - get a patient's human papillomavirus (`HPV`) immunisation history, based on their NHS number
 
     You cannot currently use this API to:
+
     - get details of other types of immunisation
 
     You get the following data:
+
     - immunisation event details
     - patient demographic details, as captured at the point of immunisation
 
-    The patient demographic details might differ from those held in the [Personal Demographics Service (PDS)](https://digital.nhs.uk/services/demographics).
-    To get demographic details from PDS, use the [Personal Demographics Service FHIR API](https://digital.nhs.uk/developer/api-catalogue/personal-demographics-service-fhir).
-    
-    If you are using this API in [user-restricted access mode](https://digital.nhs.uk/developer/guides-and-documentation/security-and-authorisation#user-restricted-apis) the patient must have their identity verified to 'high' (P9) level.
-    
+    The patient demographic details might differ from those held in the Personal Demographics Service (PDS). To get demographic details from PDS, use the Personal Demographics Service FHIR API.
```

Comment:
@nathanwall please update` Personal Demographics Service FHIR API` to
`[Personal Demographics FHIR API](https://digital.nhs.uk/developer/api-catalogue/personal-demographics-service-fhir)`